scipy/sparse/__init__.py

```diff
@@ -230,7 +230,6 @@
 
 >>> A = lil_array((1000, 1000))
 >>> A[0, :100] = rand(100)
->>> A[1, 100:200] = A[0, :100]
```

Comment:
@dschult had to remove this line because otherwise it fails and compains about 1D slices.